torch/utils/data/dataloader.py

```diff
@@ -567,20 +572,26 @@ def _get_shared_seed(self):
                 ws = dist.get_world_size()
                 store = dist.distributed_c10d._get_default_store()
                 if rank == 0:
-                    store.set("_dl_shared_seed", str(_shared_seed))
+                    _shared_seed_str = str(_shared_seed)
+                    store.set("_dl_shared_seed", _shared_seed_str)
```

Comment:
super nit: keys might be module level constants

Reply:
Sounds great. Will change it.